mayastor-test/e2e/common/io_connect_node.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Script to disconnect a node from another node using iptables
+# $1 is the node-name to isolate/restore
+# $2 is the other node-name
+# $3 is "I" to disconnect, "D" to reconnect
```

Comment:
From the iptables point of view "I" (Insert) and "D" (Delete) makes sense but for the user of the script "D" (Disconnect) "R" (Reconnect) might be more intuitive.